ecl/eclccserver/eclccserver.cpp

```diff
@@ -849,6 +849,43 @@ static void removePrecompiledHeader()
 // Class EclccServer manages a pool of compile threads
 //------------------------------------------------------------------------------------------------------------------
 
+static StringBuffer &getQueues(StringBuffer &queueNames)
```

Comment:
Feels like this should be shared code?

Reply:
not sure. NB: this is moved code was in EclccServer::run()
Could be a moved to a shared place, but don't know anywhere else that wants a named eclccserver's filtered queue list at the moment.